### ISSUE TITLE
Labels size 

### ISSUE THREAD

**Issue by simonebonato** [SimoPippa/face_recognition#1]
I can't find a way to make the predicted labels of a proper size when plotting the predictions at the end...

I tried many different solutions but since the image dimension varies, as well as the face dimensions, sometimes the values are fine, but others are not...

Some help would be appreciated since I am not really super good with plots and objects.

PS: I tried to load the model with pickle, but without success because the model i pretrained and is saved in a Keras format (I think)
